core/src/apps/cardano/__init__.py

```diff
@@ -6,6 +6,9 @@
 CURVE = "ed25519"
 SEED_NAMESPACE = [HARDENED | 44, HARDENED | 1815]
 
+MAINNET_PROTOCOL_MAGIC = 764824073
```

Comment:
is that really shelley mainnet protocol magic as well?

Reply:
If not or we are not sure, I would rather remove this and retain only the testnet magic (42 apparently) - it's not a breaking thing and better to not mislead users into thinking they are interacting with the mainnet if they actually aren't

Reply:
ok, agreed offline that this will be changed in later PRs anyway